doc/users/next_whats_new/2020-05-tac.rst

```diff
@@ -0,0 +1,23 @@
+Add patch-work inspired API for generating grids of Axes
+--------------------------------------------------------
+
```

Comment:
Please clearly mark as Experimental

Reply:
and also make explicit that this is Semantic/named axes layouting (basically we want even non-R folks to grok this immediately)